source/DomainTheory/index.lagda

```diff
@@ -45,6 +46,9 @@ module DomainTheory.index where
 6. Useful facts about sup-complete dcpos, e.g. the directification of families
 7. Definitions and basic properties of the way-below relation, including
    compactness
+
+(B1)─(B4) These files are due to Brendan Hart and were ported to TypeTopology by
+          Martín Escardó, cf. PCF.Lmabda.index.lagda.
```

Comment:
There is a typo in line 50.

Reply:
Sorry, but I can't seem to spot the typo. What is it?

Reply:
`Lmabda`.

Reply:
Ah, I was only looking at line 50, not 51 :joy: